sonar-duplications/src/main/java/net/sourceforge/pmd/cpd/SourceCode.java

```diff
@@ -36,7 +36,9 @@
  * <p>Not intended to be instantiated by clients.</p>
  *
  * @since 2.2
+ * @deprecated since 5.5
  */
+@Deprecated
 public class SourceCode {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)